requirements.txt

```diff
@@ -1,73 +1,16 @@
-alembic==1.4.2
-aniso8601==3.0.0
-appdirs==1.4.3
 APScheduler==3.5.1
-asn1crypto==0.24.0
-attrs==19.3.0
-awsebcli==3.14.1
-bcrypt==3.1.7
-black==19.3b0
-blessed==1.15.0
+awsebcli==3.19.0
```

Comment:
Why do we need this?

Reply:
This later version of awsebcli?

Reply:
Where do we use `awsebcli` though?

Reply:
Although I don't know much about it but it is used in deploy.sh file

Reply:
@SanketDG  https://github.com/anitab-org/mentorship-backend/blob/6fe6929fed2007ed43242f36c44f9bcca1d79979/deploy.sh#L33-L37

Reply:
I still think it can be removed, @isabelcosta do you have any thoughts on this?

I am guessing deploying to AWS in itself a very exclusive thing so the person itself can do it, and it's not in the stricter sense, a dependency.

Reply:
True it's not a dependency in the stricter sense. That script though, when building the app (previous Travis CI) this script would run on TravisCI and deploy the app on AWS. So once we have AWS access that will be useful again!? @SanketDG

Reply:
@isabelcosta @SanketDG should it be removed for now?

Reply:
> So once we have AWS access that will be useful again!?

I see, we could also use a separate file for those like `dev-requirements.txt`, let's go ahead with this @KapilBansal

Reply:
@SanketDG Okay, I am removing awsebcli from requirements and made a new dev-requirements for that.